client-test/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM openjdk:8u212-jre-slim-stretch
+ENV KELPIE_VERSION 1.1.0
+
+RUN apt-get update && apt-get install wget unzip -y && \
```

Comment:
Let's use gradle shadowJar.
I think it should work.
```
docker {
    files tasks.shadowJar.outputs
}
```
Then, Dockerfile can get all the jars including kelpie.

Reply:
done, it works